src/components/SearchResultListing/index.js

```diff
@@ -131,34 +158,14 @@ class SearchResultListing extends PureComponent {
                 )}
 
                 <td className="text-right">
-                  <ButtonGroup bsSize="sm" className="crud--search-result-listing__action-buttons">
-                    <Button onClick={this.handleShow(instance, index)}>
-                      <Glyphicon glyph='glyphicon-eye-open' />
-                      {' '}
-                      {i18n.getMessage('crudEditor.show.button')}
-                    </Button>
-
-                    <Button onClick={this.handleEdit(instance, index)}>
-                      <Glyphicon glyph='edit' />
-                      {' '}
-                      {i18n.getMessage('crudEditor.edit.button')}
-                    </Button>
-
-                    <ConfirmDialog
-                      trigger='click'
-                      onConfirm={this.handleDelete(instance)}
-                      title='Delete confirmation'
-                      message={i18n.getMessage('crudEditor.delete.confirmation')}
-                      textConfirm={i18n.getMessage('crudEditor.delete.button')}
-                      textCancel={i18n.getMessage('crudEditor.cancel.button')}
-                    >
-                      <Button>
-                        <Glyphicon glyph='trash' />
-                        {' '}
-                        {i18n.getMessage('crudEditor.delete.button')}
-                      </Button>
-                    </ConfirmDialog>
-                  </ButtonGroup>
+                  <SearchResultButtons
+                    model={this.props.model}
+                    instance={instance}
```

Comment:
No need for "instance" and "index" params.  Send called handlers instead, like `onShow={this.handleShow(instance, index)}`

Reply:
Done.